extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/V2JdbcTableCatalogPrivilegesBuilderSuite.scala

```diff
@@ -89,4 +89,56 @@ class V2JdbcTableCatalogPrivilegesBuilderSuite extends V2CommandsPrivilegesSuite
       }
     }
   }
+
+  test("Extracting table info with ResolvedTableTableExtractor") {
+    val ns1 = "testns1"
+    val tbl = "testtbl"
+    withDatabase(s"$ns1") { ns =>
+      sql(s"CREATE NAMESPACE $ns")
+      withTable(s"$catalogV2.$ns1.$tbl") { t =>
+        sql(s"CREATE TABLE IF NOT EXISTS $t(key int)")
+        Seq(
+          s"SHOW CREATE TABLE $t").foreach { str =>
```

Comment:
Are you planning more test cases here?

Reply:
It is good enough to cover all 3 table extractors for datasource v2 logical plans, including `ResolvedTable` and `DataSourceV2Relation` in this PR. For the rest in v1, the `catalog `of `Table` remains to be None by default.

Reply:
Thus, let's avoid his seq?